broker-daemon/proto/broker.proto

```diff
@@ -1311,6 +1311,8 @@ message Balance {
   string total_pending_channel_balance = 4;
   // Balance in the listed currency that is currently in pending closing channels and unconfirmed wallet balance. This is currently in common units (e.g. 0.5 BTC)
   string uncommitted_pending_balance = 5;
+  // Total balance across all channels that are reserved as fees due to a force close. This is currently in common units (e.g. 0.5 BTC)
+  string total_commit_fees = 6;
```

Comment:
Is there a different name we could give this? like `total_reserved_channel_balance` or something? Commit fee is not very descriptive and feels very LND specific. What we would like the name to communicate is that this is part of the channel balance that is reserved for channel force closes.